### PR TITLE
Add uniqueKeyPolicy to container

### PR DESCRIPTION
**Pull request by southpolesteve**

### CODE REVIEW

**Comment by christopheranderson** [comment]
For v3, we'll eventually introduce a builder pattern for this, but this allows portal to be unblocked and easy backporting to master, so this is perfect.

File: src/client/Container/UniqueKeyPolicy.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export interface UniqueKeyPolicy {
```
Comment (christopheranderson) [inline src/client/Container/UniqueKeyPolicy.ts]: We should have TSDocs for public types.

Comment (christopheranderson) [inline src/client/Container/UniqueKeyPolicy.ts]: Don't you need to update ./index.ts to export this type to avoid weird import paths?

Comment (southpolesteve) [reply src/client/Container/UniqueKeyPolicy.ts]: Yep. Added.

**Pull request merged**